Ability to create non-translatable attributes fixtures + info about non-translatable attributes. (#12256)
Sylius/Sylius: docs/book/products/attributes.rst

- datetime - accurate date and time of an event
- select - genre(s) of a book. one or more of them can be selected

Non-translatable attribute
---------

> Some attributes shouldn't be changed in different locales (like numbers, dates), so by disabling it's translatability there's no need to write translation for all locales and write it once for global translation.

Reviewer: ```suggestion
Some attributes (dates, author name) don't need a different value in each locale. For those attributes, we introduced the possibility to disable translation.  Shop Owner declares values only once and regardless of the chosen locale customer will see a proper attribute value.
```
?

** Warning **

> Once attribute has disabled translatability it will erase all locales for this attribute.

Reviewer: ```suggestion
Once the attribute has disabled translatability it will erase attribute values all locales for this attribute.
```
?

- datetime - accurate date and time of an event
- select - genre(s) of a book. one or more of them can be selected

Non-translatable attribute
--------------------------

Some attributes (dates, author name) don't need a different value in each locale. For those attributes, we introduced the possibility to disable translation.
Shop Owner declares values only once and regardless of the chosen locale customer will see a proper attribute value.

> Warning
> 
> Once the attribute has disabled translatability it will erase attribute values in all locales for this attribute.

Reviewer 2: ```suggestion
.. warning::

        Once the attribute has disabled translatability it will erase attribute values in all locales for this attribute.
```
